### ISSUE TITLE
Instead of the normal rectangle it just started doing rectangle but with the edges rounded.

### ISSUE THREAD

**Issue by Kazuki70** [ShareX/ShareX#6391]
No it's not this and this didn't fix it. https://github.com/ShareX/ShareX/issues/1778

Here's an image:
![image](https://user-images.githubusercontent.com/98607095/179793141-2786cdb8-ba0b-4e66-aa11-c67676afd140.png)

Edited because the color of the picture made it hard to notice.

**Comment by Kazuki70** [comment]
OK! I finally decided to spend a good portion of my time on this and fixed it.
Basically, it's just the weirdest default configuration I've seen on a software in a long time...
I had to download the latest version to even fix it anyway.
After you click to make a new screen capture, before you make the capture, you have to click on Tool Options icon and then make the Corner Radius "0", it comes with like 10 by default.
By the way, everyone at work and some people I play online with have already stopped using this software, all because of that.
No idea if they will want to go back to this one now since the other software works fine for them.
Don't ruin it with things like this again pretty please.

**Comment by Jaex** [comment]
> By the way, everyone at work and some people I play online with have already stopped using this software, all because of that. No idea if they will want to go back to this one now since the other software works fine for them.

By default region corner radius is "0". Only reason you had rounded corners is because you changed the setting yourself. So you are clearly lying here telling that everyone at your work and your friends stopped using ShareX because of that. Which is kinda hilarious how you could easily tell lie about such thing.

**Comment by Kazuki70** [comment]
Whats hilarious is your response.
What the hell kind of person assumes stupid stuff like that...
Are you the dev and seriously that hard headed? Fix yourself and then come help people. What the hell did I just read?.......
Every PC at the office had Lightshot, I put your software on every pc, everyone liked it, then overtime every single PC got this problem, it was probably an update.
If you read everything you would've known that I couldn't even change it, the option wasn't there and I never had any reason to even open the options, it was all new to me when I posted the Issue.
On my discord everyone got this too.C
It was like 9 out of 11 people who told me, they even complained to me as if I had to know how to fix it.L
I am installing Lightshot in the office again as I write this line.O
I'll just "lie" and tell them that the app is too prone to issues and the support sucks.W
Thanks for the greaat attitude. Your software is perfect, don't even read the Issues, no need.N

**Comment by Kazuki70** [comment]
> > By the way, everyone at work and some people I play online with have already stopped using this software, all because of that. No idea if they will want to go back to this one now since the other software works fine for them.
> 
> By default region corner radius is "0". Only reason you had rounded corners is because you changed the setting yourself. So you are clearly lying here telling that everyone at your work and your friends stopped using ShareX because of that. Which is kinda hilarious how you could easily tell lie about such thing.

Whats hilarious is your response.
What the hell kind of person assumes stupid stuff like that...
Are you the dev and seriously that hard headed? Fix yourself and then come help people. What the hell did I just read?.......
Every PC at the office had Lightshot, I put your software on every pc, everyone liked it, then overtime every single PC got this problem, it was probably an update.
If you read everything you would've known that I couldn't even change it, the option wasn't there and I never had any reason to even open the options, it was all new to me when I posted the Issue.
On my discord everyone got this too.C
It was like 9 out of 11 people who told me, they even complained to me as if I had to know how to fix it.L
I am installing Lightshot in the office again as I write this line.O
I'll just "lie" and tell them that the app is too prone to issues and the support sucks.W
Thanks for the greaat attitude. Your software is perfect, don't even read the Issues, no need.N
